assets/src/dashboard/utils/index.js

```diff
@@ -14,14 +14,15 @@
  * limitations under the License.
  */
 
+export { default as useResizeEffect } from '../../edit-story/utils/useResizeEffect';
 export { default as clamp } from './clamp';
 export { default as getCurrentYAxis } from './getCurrentYAxis';
 export { default as getFormattedDisplayDate } from './getFormattedDisplayDate';
 export { default as groupBy } from './groupBy';
 export { default as keyboardOnlyOutline } from './keyboardOnlyOutline';
-export { default as useFocusOut } from './useFocusOut';
+export { default as memoize } from './memoize';
+export { default as throttleToAnimationFrame } from './throttleToAnimationFrame';
 export { default as useDashboardResultsLabel } from './useDashboardResultsLabel';
+export { default as useFocusOut } from './useFocusOut';
 export { default as usePagePreviewSize } from './usePagePreviewSize';
 export { default as useStoryView } from './useStoryView';
-export { default as throttleToAnimationFrame } from './throttleToAnimationFrame';
-export { default as useResizeEffect } from '../../edit-story/utils/useResizeEffect';
```

Comment:
just added `memoize` and then vscode organized imports here (`option + shift + o`)